addons/ccm-hetzner/ccm-hetzner.yaml

```diff
@@ -31,15 +31,17 @@ spec:
   revisionHistoryLimit: 2
   selector:
     matchLabels:
-      app: hcloud-cloud-controller-manager
+      app.kubernetes.io/instance: 'hccm'
+      app.kubernetes.io/name: 'hcloud-cloud-controller-manager'
```

Comment:
This requires migration because PodSelector labels are immutable. See #2824

Reply:
Added migration path